render/render.js

```diff
@@ -885,6 +885,10 @@ module.exports = function($window) {
 		vnode.dom = old.dom
 		vnode.domSize = old.domSize
 		vnode.instance = old.instance
+		vnode.text = old.text
+		vnode.children = old.children
+		vnode.state = old.state
```

Comment:
Keep this line and drop the rest. I specifically *want* the new attributes and children - having "almost always the new attributes but sometimes in obscure cases different" is a really bad idea IMHO. (This is in direct contradiction to [this comment](https://github.com/MithrilJS/mithril.js/issues/2067#issuecomment-449178876).)